plugins/remote-config/api/api.js

```diff
@@ -338,7 +338,7 @@ plugins.setConfigs("remote-config", {
         var maximumParametersAllowed = plugins.getConfig("remote-config").maximum_allowed_parameters;
         var maximumConditionsAllowed = plugins.getConfig("remote-config").conditions_per_paramaeters;
         var collectionName = "remoteconfig_parameters" + appId;
-
+        parameter.ts = Date.now();
```

Comment:
Is this ts used as the created timestamp?

Reply:
yes, showing "-" for those